posix.mak

```diff
@@ -383,7 +383,8 @@ clean :
 	rm -rf $(ROOT_OF_THEM_ALL) $(ZIPFILE) $(DOC_OUTPUT_DIR)
 
 zip :
-	zip $(ZIPFILE) $(MAKEFILE) $(ALL_D_FILES) $(ALL_C_FILES) index.d win32.mak win64.mak osmodel.mak
+	-rm -f $(ZIPFILE)
+	zip -r $(ZIPFILE) . -x .git\* -x generated\*
```

Comment:
And from what we know he uses it to sync dev work between different PCs (b/c for some weird reason installing git is a hassle), so it's actually beneficial to copy all files.
An even better idea would be to teach Walter a different workflow, e.g. `git daemon` or `rsync`, and getting rid of the zip targets alltogether.